packages/IMAPClient-Core.package/ICEmail.class/instance/readBody.st

```diff
@@ -0,0 +1,7 @@
+accessing
+readBody
+
+	(body isNil or: [body isEmpty]) ifTrue: [
+		self retrieveBody].
+	^body.	
```

Comment:
space between ^ and body

Reply:
How about `self body isEmpty ifTrue: [self retrieveBody]`?

Reply:
never mind, readBody is changed in your latest commit: https://github.com/hpi-swa-teaching/IMAPClient/pull/398/commits/126218b6904e436535a5ba19097dfb0a856a211b